src/torrent/bitfield.h

```diff
@@ -50,8 +50,10 @@ class LIBTORRENT_EXPORT Bitfield {
   typedef value_type*           iterator;
   typedef const value_type*     const_iterator;
 
-  Bitfield() : m_size(0), m_set(0), m_data(NULL)    {}
+  Bitfield() = default;
   ~Bitfield()                                       { clear(); }
+  Bitfield(const Bitfield&) = delete;
+  Bitfield& operator=(const Bitfield&) = delete;
```

Comment:
Make a separate PR with just these changes to ctor, deleted operators and default member variable values.

Also respect empty newlines, don't pack different categories of member functions together like that.